style.css

```diff
@@ -1,7 +1,84 @@
+.hiddenHTML {
+  display: none;
+}
+
 #lyngkboard {
   border: 2px solid black;
+  position: absolute;
+  top: 10%;
+  left: 20%;
+}
+
+#unclaimedCanvas {

```

Comment:
Try allowing the divs to float.  The layout you are trying to get has the "unclaimed colors" on the left of the "board" and then on the right we have a stack of "board title", "player to move", "move list", "first player's colors", "second player's colors".  I still don't see where "number of stacks claimed" went...

https://www.w3schools.com/cssref/pr_class_float.asp